### ISSUE TITLE
Max poor grass rendering distance

### ISSUE THREAD

**Issue by wizardofid29** [TheGameCreators/GameGuruRepo#1786]
Grass when looking up hill is not being rendered far enough, which results in issues like the above picture.

Remedy
Grass should have rendering distance slider, to compensate, would allow higher end systems to have longer draw distances for grass

![grass](https://user-images.githubusercontent.com/82841733/149807672-bf1152bb-d970-46e0-b25c-22043f3e2768.jpg)

**Comment by UltraVox001** [comment]
Yes. It would also be necessary to do with the grass, the same thing as with the trees. It takes impostor grasses. Very distant grasses (beyond the max setting you rightly ask for), don't need to be animated. They must be simulated in 2D (immobile) to give the impression that the vegetation continues into the distance. Grass animations should start when the player approaches. It radiates an activation zone around it.

**Comment by wizardofid29** [comment]
Problem is they have been aware of this for months now, and still hasn't
been addressed.Whether alpha or not is not the point.This is some thing
that should have been fixed when they were working on it.Sort of half baked
feature.

Terrain textures will not go down well, with users in the march
release.With the release around the corner.It is super critical that things
like this get addressed as well as mentioned that it receives the TLC it
needs.

This definitely not some thing that should be done after the march
release.It is currently the weakest point in the visuals of max overall.
Default terrain textures just isn't there and being the first thing users
will see as first impression, iam quite concerned.

**Comment by UltraVox001** [comment]
Like you, I am also worried. I don't know why this work hasn't been done yet. Unmodifiable textures (strange!). Vegetation in 2D Sprites, instead of using textured 3D models in PBR, like in all modern FPS. Yes, I understand your concerns. I have the same. In March 2022, if things are still as they are, people are going to have a good laugh watching Max's videos. Because there will be nothing interesting to see. What we will see is what we already saw in video games that date back 15 years.

I wish something better for Max and for TGC. They have the opportunity to make a difference. If they don't by March, there will be very bad publicity.

**Comment by rvanner** [comment]
This week we will be adding in all new grass veg models that match perfectly with each texture type. So please re-review this once it's available.
